Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -90,6 +91,18 @@ def check_clt_up_to_date
         EOS
       end
 
+      def check_xcode_8_without_clt_on_el_capitan
+        return unless MacOS::Xcode.without_clt?
+        # Scope this to Xcode 8 on El Cap for now
+        return unless MacOS.version == :el_capitan && MacOS::Xcode.version >= "8"
+
+        <<-EOS.undent
+          You have Xcode 8 installed without the CLT; this causes certain builds to fail.
+          Please install the CLT via:
+            xcode-select --install
```

Comment:
It's totally inconsistent now I look at it - we have messages using both with and without. 😰 I'll update this now, and make a PR to update other parts for consistency later.